plugins/tiddlywiki/dynannotate/examples/simple.tid

```diff
@@ -10,75 +10,68 @@ caption: Simple
 $example$
 \end
 
-<div class="tc-dynannotation-example-info">
-
-!! Simple annotation
-
-We use the `target*` attributes to specify a target string for the annotation and optionally a prefix and suffix for disambiguating multiple occurances.
-
-</div>
-
-<<show-example """
+\define search-example-code()
+\rules only filteredtranscludeinline
 <$dynannotate
-	target="the"
-	targetPrefix="Yet "
-	targetSuffix=" speed"
+	search="{{{ [{$:/temp/dynannotate-example/search}!is[blank]else[The human mind]] }}}"	{{{ [{$:/temp/dynannotate-example/searchMode}!search::some[literal normal]addprefix[
+	searchMode="]addsuffix["]] }}}
+	searchCaseSensitive="{{{ [{$:/temp/dynannotate-example/searchCaseSensitive}] }}}"	{{{ [{$:/temp/dynannotate-example/searchClass}!is[blank]addprefix[
+	searchClass="]addsuffix["]] }}}
 >
 <$transclude tiddler="$:/plugins/tiddlywiki/dynannotate/example-text-1" mode="block"/>
+<$transclude tiddler="$:/plugins/tiddlywiki/dynannotate/example-text-2" mode="block"/>
 </$dynannotate>
-""">>
+\end
 
 <div class="tc-dynannotation-example-info">
 
-!! Plain text searching
+!! Annotations with `target`
 
-We use the `search` attribute to specify a search string for highlighting:
+We use the `target*` attributes to specify a target string for the annotation and optionally a prefix and suffix for disambiguating multiple occurances.
 
 </div>
 
-<<show-example """
-<$dynannotate
-	search="the"
+<<show-example """<$dynannotate
+	target="the"
+	targetPrefix="Yet "
+	targetSuffix=" speed"
 >
 <$transclude tiddler="$:/plugins/tiddlywiki/dynannotate/example-text-1" mode="block"/>
 </$dynannotate>
 """>>
 
-<div class="tc-dynannotation-example-info">
-
-!! Regular expression searching
 
-We use the `mode` attribute set to `regexp` to highlight matches of a regular expression:
 
-</div>
-
-<<show-example """
-<$dynannotate
-	search="the|an"
-	searchMode="regexp"
-	searchClass="tc-dynannotation-search-overlay-blurred"
->
-<$transclude tiddler="$:/plugins/tiddlywiki/dynannotate/example-text-1" mode="block"/>
-</$dynannotate>
-""">>
 
 <div class="tc-dynannotation-example-info">
 
-!! Normalised whitespace searching
+!! Highlights with `search`
 
-We use the `mode` attribute set to `whitespace` to search for a string with whitespace normalised (ie runs of whitespace are collapsed to a single space for matching purposes):
+We use the `search` attribute to specify a search string for highlighting.<br>
```

Comment:
What is the purpose of using `<br>` here? Why can't we rely on ordinary paragraph formatting?

Reply:
I thought the space between the lines was too large with paragraphs. Changed it, though.